src/main/java/com/mongodb/jdbc/BsonTypeInfo.java

```diff
@@ -23,8 +25,58 @@
     public static final String SYMBOL_TYPE_NAME = "symbol";
     public static final String TIMESTAMP_TYPE_NAME = "timestamp";
     public static final String UNDEFINED_TYPE_NAME = "undefined";
+    public static final String NULL_TYPE_NAME = "null";
     public static final String BSON_TYPE_NAME = "bson";
 
+    public static Pair<String, Integer> getBsonTypeNameAndNullability(
```

Comment:
This method is 90% the same as the `MongoSQLColumnInfo` constructor. The differences are subtle, but I figured for now we'd just stick with the duplication and have the refactor ticket in the scheduled queue address unification.

Reply:
Agreed. Unification should be easy, but I like it in a separate ticket.

Reply:
Actually, I expected to use the `ResultSetMetadata` here, but this is good for now.